src/pages/meditation/Meditation.tsx

```diff
@@ -0,0 +1,31 @@
+import MeditationTimer from '@pages/meditation/components/MeditationTimer';
+import MeditationCounter from '@pages/meditation/components/MeditaionCounter';
+import MeditationLabel from '@pages/meditation/components/MeditationLabel';
+import { Container } from '@pages/meditation/Meditation.style';
+import Confirm from '@components/Confirm';
+import { useRecoilState } from 'recoil';
+import { endButtonPushed } from '@pages/meditation/components/MeditationEndButton';
+
+export const Meditation = () => {
+  const [confirmCaptured, _] = useRecoilState(endButtonPushed);
```

Comment:
값만 사용하시려면 이렇게도 할 수 있을거 같아요
```suggestion
  const confirmCaptured = useRecoilValue(endButtonPushed);
```